include/rocksdb/options.h

```diff
@@ -2289,13 +2289,17 @@ struct CompactionServiceOptionsOverride {
   Env* env = Env::Default();
   std::shared_ptr<FileChecksumGenFactory> file_checksum_gen_factory = nullptr;
 
+  // DEPRECATED - CFOption pointer configurations will be passed as static name
+  // These will need to be registered in the remote worker's ObjectRegistry,
+  // then created by Type::CreateFromString() when needed.
```

Comment:
This remove compaction APIs are experiment, would it be safe to just remove these?

Reply:
There's an old prototype code in fbcode still referencing this, so that's why I didn't remove them at first, but I guess can clean up the old prototype now before we merge this PR into fbcode.